portfolio/src/main/webapp/map.js

```diff
@@ -0,0 +1,21 @@
+var position = {lat: 40.7, lng: -74};
+
+//Gets a set of latitude, longitude.
+function grabPosition(userLat, userLng){
+    position["lat"] = userLat;
+    position["lng"] = userLng;
+    console.log(position["lat"]);
+    console.log(position["lng"]);
+    localStorage.setItem("userLat", position["lat"]);
+    localStorage.setItem("userLng", position["lng"]);
+}
+
+//Sets latitude and longitude.
+function giveLatitude(){
+    setLat = parseFloat(localStorage.getItem("userLat")); //TO-DO: Find a way to store values that isn't local.
```

Comment:
I imagine this'll be in some datastore entity that represents an event eventually, at least when we're looking at an event's location. Maybe the user's location will always be in local storage?

Reply:
Good point- I'll look into datastore for storing event coordinates (and the next feature for displaying path coordinates).